x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.js

```diff
@@ -154,11 +166,7 @@ function DetailView({ errorGroup, urlParams, location }) {
         </DiscoverButton>
       </HeaderContainer>
 
-      <ContextProperties
-        timestamp={timestamp}
-        url={url}
-        stickyProperties={stickyProperties}
-      />
+      <StickyProperties stickyProperties={stickyProperties} />
```

Comment:
StickyProperties component now only takes a list of properties, so everything should be declared in that structure (previously url and timestamp was passed in separately).